source/MaterialXFormat/Util.cpp

```diff
@@ -234,6 +234,86 @@ void flattenFilenames(DocumentPtr doc, const FileSearchPath& searchPath, StringR
     }
 }
 
+FileSearchPath getResolvedDefinitionPath(const FileSearchPath& userDefinitionPath, bool includeSubFolders)
+{
+    FileSearchPath resolvedDefinitionPath;
```

Comment:
This doesn't seem correct based on the path logic we worked out.
It does not include the core paths nor MATERIALX_SEARCH_PATH.